src/test-resources/generator/package.json

```diff
@@ -15,9 +15,9 @@
     "@types/node": "^12.7.7",
     "@types/puppeteer": "^1.19.1",
     "axe-cli": "3.2.0",
-    "axe-core": "3.5.4",
+    "axe-core": "3.5.5",
     "axe-puppeteer": "^1.0.0",
-    "puppeteer": "^1.19.0",
```

Comment:
is this puppeteer update intended?

Reply:
I included it because updating it was necessary to regenerate the new test files against current chrome stable (the old version was outdated enough that it refused to connect to current Chrome). I left that as part of the same PR so we can keep the version the test cases were generated against in sync